recipes/ipycache/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: ipycache
+  version: {{ version }}
+
+source:
+  fn: ipycache-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/i/ipycache/ipycache-{{ version }}.zip
+  md5: c78c6094ffab45b0e33c2f556705fa11
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - ipython
+
+test:  
+  commands:
+    - curl -s -L https://github.com/rossant/ipycache/tarball/master -o master.tgz
+    - mkdir source
+    - cd source
+    - tar --strip-components=1 -xvzf ../master.tgz
+    - nosetests 
+    - python ipynb_runner.py -v -s examples/example.ipynb  # [not win]
+    - python ipynb_runner.py -v -s examples/example_outputs.ipynb  # [not win]
+  
+  requires:
+    - nose
+    - numpy
+    - matplotlib
+    - ipython
+    - nbformat
+    - ipykernel
+
+about:
+  home: http://packages.python.org/ipycache
+  license: BSD-like license
+  license_file: LICENSE.md
```

Comment:
As mentioned, in [gitter]( https://gitter.im/conda-forge/conda-forge.github.io?at=5745af0010f0fed86f47fd79 ) please feel free to include the license file with the recipe.